src/alfasim_score/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
```

Comment:
As we agreed, delete the content of this file and mport the fixture for now.
Refactoring of the structure folders and import of the fixture here will be done in PWPA-1963

Reply:
Wait, based on https://github.com/ESSS/alfasim-score/pull/2#discussion_r1644879917, I understood it would be here.

Reply:
O talked to Gabriel because I think it is better that `conftest` to be like: 
```
pytest_plugins = [
    'alfasim_score.fixtures',
]
```
instead of having the fixtures. 
And I asked him to change the folders structure like this
![image](https://github.com/ESSS/alfasim-score/assets/27425264/6889cd9f-58c1-4b47-b59d-2815276e121d)

However, he thinks it is better to do in another task.

Reply:
The `python` folder was missing, thus `conftest.py` was in the wrong place. Also, `fixtures.py` should be in `alfasim_score` folder.